### PR TITLE
MASTER W83-1496/CHARGE-STATUS-FILTER-MASTER

### PR DESCRIPTION
**Pull request by IslamMesha**
- Apply the CHARGE_STATUS filter in export orders

### CODE REVIEW

**Review by essam5 — APPROVED** [review]

File: datamigration/graphql/filters.py [hunk 1]
```diff
@@ -20,7 +19,7 @@ def filter_order_search(qs, _, value):
     return qs
 
 
-def filter_payment_type(qs, field, value):
+def filter_payment_type(qs, _, value):
```
Comment (Ashraf-Khaled-BE) [inline datamigration/graphql/filters.py:22]: add type lint as the change here is vague

**Review by Ashraf-Khaled-BE — CHANGES_REQUESTED** [review]

File: datamigration/graphql/filters.py (hunk 1 above)
Comment (IslamMesha) [reply datamigration/graphql/filters.py:22]: Check this file in the core -> saleor/graphql/account/filters.py this is a standard they follow. @Ashraf-Khaled-BE

Comment (Ashraf-Khaled-BE) [reply datamigration/graphql/filters.py:22]: i didn't say its wrong , I said its vague. for any new dev will join us , I wont tell him please go check the core buddy , so please its our plugin now not the core , add linting and docstring for the function

**Review by Ashraf-Khaled-BE — CHANGES_REQUESTED** [review]
Docstring and lint need to be added

**Review by Ashraf-Khaled-BE — APPROVED** [review]

**Pull request merged**